src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/softserve/models/User.java

```diff
@@ -0,0 +1,40 @@
+package com.softserveinc.ita.dkrutenko.pageobjects.softserve.models;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class User {
```

Comment:
add @Builder here, and after this, you would be able to delete constructor and build the object in repo class in this way

User.builder().firtstName.build();

Reply:
i dont know how to realize this...

Reply:
it should be after constructor ?

Reply:
done